src/factors/flux/MixtureFluxPose2Pose2.jl

```diff
@@ -86,66 +90,67 @@ end
 
 function calcVelocityInterPose2!( nfb::MixtureFluxPose2Pose2,
                                   iPts::AbstractMatrix{<:Real},
```

Comment:
Also think these should be `AbstractVector`

Reply:
Ok, I'll change it. It will probably change with Manifold integration